package.json

```diff
@@ -44,15 +44,14 @@
     "http-status-codes": "^2.2.0",
     "launchdarkly-js-client-sdk": "^2.24.2",
     "lodash": "^4.17.21",
-    "lodash.omit": "^4.5.0",
```

Comment:
Lodash's recommendation is to no longer use the per-method packages so I removed this and simply use `omit` from the main lodash package.